dotenv-linter/tests/common/test_dir.rs

```diff
@@ -202,7 +215,11 @@ impl TestDir {
         )
     }
 
-    fn init_cmd() -> Command {
-        Command::cargo_bin(env!("CARGO_PKG_NAME")).expect("command from binary name")
+    fn init_cmd(&self) -> Command {
+        let mut cmd = Command::cargo_bin(env!("CARGO_PKG_NAME")).expect("command from binary name");
+
+        cmd.envs(&self.envs.clone().unwrap_or_default());
```

Comment:
Doesn't it work with reference?
Something like that `self.envs.as_ref().unwrap_or(&[])`

Reply:
And take a look at CI warnings please.

Reply:
Ah, I wasn't familiar with `as_ref()`, this should do it. https://github.com/dotenv-linter/dotenv-linter/pull/625/commits/9eeead1552b1a18bc839726f37ba7ab5f2305f36
Also ran the formatter as well 👌